zcash_client_sqlite/src/scan.rs

```diff
@@ -57,6 +57,38 @@ struct WitnessRow {
 pub fn scan_cached_blocks<P: AsRef<Path>, Q: AsRef<Path>>(
     db_cache: P,
     db_data: Q,
+) -> Result<(), Error> {
+    scan_cached_block_batch(db_cache, db_data, std::i32::MAX)
+}
+
+/// Scans new blocks added to the cache for any transactions received by the tracked
+/// accounts but does so in batches of limited size.
```

Comment:
```rust
/// Scans at most `limit` new blocks added to the cache for any transactions received by
/// the tracked accounts.
///
/// This function will return without error after scanning at most `limit` new blocks, to
/// enable the caller to update their UI with scanning progress. Repeatedly calling this
/// function will process sequential ranges of blocks, and is equivalent to calling
/// `scan_cached_blocks` once on the entire cache.
```

Also alter the documentation for the original function to read `Scans all new blocks...`

Reply:
Incorporated suggested wording in documentation.

Addressed in ab100d3b35e2130e0b81631cd20585cde2ef9f75